src/challenges/challengeOne/5.Modal/Modal.test.tsx

```diff
@@ -1,8 +1,71 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Modal } from "./Modal";
-
-// describe("Modal", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Modal, ModalProps } from "./Modal";
+
+const testHeaderText = "Modal Header";
+const testChildren = <p>Modal content</p>;
+
+describe("Modal", () => {
+  const renderComponent = (props: ModalProps) => {
+    render(<Modal {...props} />);
+  };
+
+  it("renders closed modal", () => {
+    renderComponent({
+      isOpen: false,
+      onClose: jest.fn(),
+      onConfirm: jest.fn(),
+      headerText: testHeaderText,
+      children: testChildren,
+    });
+
+    const modalContainer = screen.queryByTestId("modal-container");
```

Comment:
We're not really interested in the modal container, I would instead test the modal header or content didn't show up.